examples/resources/datadog_restriction_policy/resource.tf

```diff
@@ -0,0 +1,10 @@
+
+# Create new restriction_policy resource
+
+
+resource "datadog_restriction_policy" "foo" {
+  bindings {
```

Comment:
@skarimo these are autogenerated right? wondering why resource_id isn't getting added here after I made it required

Reply:
The generator is ran once to generate the boiler code. Any changes afterwards will need to be done manually